### PR TITLE
Dashboard

### LINKED ISSUE #391: [wishlist] Better distinction between own maps and others in "My maps"
"My maps" barely separates : 
- maps you own (created or cloned by you)
- maps you're allow to edit (because you were invited to)

As for now, you only see the difference when you look at the "by User" mention. 

It would be clearer if there was some sort of visual way to distinguish between the 2 cases. 

Like a different tab for maps that ain't 100% "yours", or a colorful tag to help the distinction.

### LINKED ISSUE #1062: Getting list of one's own maps?
Hello,

Unless I missed it, Umap doesn't seem to provide a way to list someone's maps.

Is there a reliable way, besides this type of (Windows) script?

```@ECHO OFF
REM Know of no other way than trial and error to find how many times to loop
FOR /L %%i IN (1,1,15) DO (
  wget -qcO - http://umap.openstreetmap.fr/en/user/joe/?p=%%i | grep -Poha "/en/map/.+?_\d+" | sed -r "s@^(.+)$@http://umap.openstreetmap.fr/\1@"
)
```

Thank  you.

--

Edit: The script above seems to work. I get the same list when scrolling manually in a browser.

--

Edit: Scrolling seems to only show the last fifty maps: Some of my maps aren't in the list :-/

### PR DESCRIPTION
**Pull request by yohanboniface**
Very basic version. Must land here so we can theme it in DSFR.

image

fix #1062
fix #391 
cf #1048

### CODE REVIEW

File: umap/static/umap/js/umap.js [hunk 1]
```diff
@@ -258,10 +254,13 @@ L.U.Map.include({
       if (this.options.onLoadPanel === 'databrowser') this.openBrowser()
       else if (this.options.onLoadPanel === 'caption') this.displayCaption()
       else if (this.options.onLoadPanel === 'datafilters') this.openFilter()
+      else if (this.options.onLoadPanel === 'share') this.renderShareBox()
```
Comment (yohanboniface) [inline umap/static/umap/js/umap.js:257]: Do we want to use `?share` instead, to be consistent with `?edit` and `?download` ?

File: umap/templates/umap/map_table.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{% load umap_tags umap_tags i18n %}
+<table class="maps">
+  <tr>
```
Comment (davidbgk) [inline umap/templates/umap/map_table.html:3]: That `tr` should be in a `thead`

File: umap/templates/umap/map_table.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    <th></th>
+    <th></th>
+    <th>{% blocktrans %}Status{% endblocktrans %}</th>
+    <th>{% blocktrans %}Last save{% endblocktrans %}</th>
+    <th>{% blocktrans %}Owner{% endblocktrans %}</th>
+    <th>{% blocktrans %}Actions{% endblocktrans %}</th>
+  </tr>
+  {% for map_inst in maps %}
```
Comment (davidbgk) [inline umap/templates/umap/map_table.html:11]: That part should be in a `<tbody>`

**Review by davidbgk — COMMENTED** [review]
I wonder if we want search results to match that new table style 🤔

File: umap/templates/umap/map_table.html (hunk 3 above)
Comment (yohanboniface) [reply umap/templates/umap/map_table.html:11]: Can you elaborate on why we should add those tags ? I thought that HTML5 thing was to only add needed tags, to KISS :)

**Comment by yohanboniface** [comment]
> I wonder if we want search results to match that new table style thinking

I'd say no, given those pages do not a address the same users/needs (we don't need status nor actions for example in the search results)

File: umap/templates/umap/map_table.html (hunk 3 above)
Comment (yohanboniface) [reply umap/templates/umap/map_table.html]: Added `<thead>` and `<tbody>`, but not sure why we'd have a `<caption>`.

Note that all seems optional according to (my understanding of) MDN: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table :)

**Comment by yohanboniface** [comment]
Merging, so I can put it in today's release, and start working on it in DSRF version. Let's iterate to make it better :)

**Comment by Pilaiv** [comment]
thank you for this new dashboard interface. I am new to github. Is here that I can suggest an addition in the "status" column, can you also add the detail "Who can see" in addition to "Who can modify"

**Comment by yohanboniface** [comment]
Makes sense!

Something like this ?

image

**Pull request merged**